mitodl/open-discussions
# Added subscribe api

#### What are the relevant tickets?
fixes https://github.com/mitodl/open-discussions/issues/112

#### What's this PR do?
creates 2 apis one is subscribe and one is un subscribe from channel

#### How should this be manually tested?
- Create 2 channels
- Listing channels http://192.168.99.100:8063/api/v0/channels/
- Unsubscribe one channel http://192.168.99.100:8063/api/v0/channels/{:url channel name}/unsubscribe/
- List channels, You wont be able to see it unsub channel
- Subscribe channel http://192.168.99.100:8063/api/v0/channels/test_channel/subscribe/
- List channel, Now you can see it.
 
@pdpinch 

#### Screenshots (if appropriate)
Output
<img width="657" alt="screen shot 2017-09-07 at 7 33 18 pm" src="https://user-images.githubusercontent.com/10431250/30168630-6f970118-9403-11e7-8ffb-2f0596151b13.png">
